include/arch/arm/arch/machine/gic_v2.h

```diff
@@ -124,13 +125,14 @@ struct gic_cpu_iface_map {
 
 extern volatile struct gic_dist_map *const gic_dist;
 extern volatile struct gic_cpu_iface_map *const gic_cpuiface;
+extern uint32_t gic_dist_config_cache[GIC_DIST_CONFIG_SIZE];
```

Comment:
There should be a comment here (as well as in the commit message) as to why the change is necessary (in my opinon).